x-pack/plugins/observability_solution/apm/server/routes/apm_routes/register_apm_server_routes.ts

```diff
@@ -94,10 +98,10 @@ export function registerRoutes({
       inspectableEsQueriesMap.set(request, []);
 
       try {
-        const runtimeType = params ? mergeRt(params, inspectRt) : inspectRt;
+        const runtimeType = params ? mergeRt(params as IoTsParamsObject, inspectRt) : inspectRt;
 
         const validatedParams = decodeRequestParams(
-          pickKeys(request, 'params', 'body', 'query'),
+          formatParams(pickKeys(request, 'params', 'body', 'query')),
```

Comment:
wdyt about getting rid of `pickKeys` here and just using a plain object?

Reply:
Sure!